modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -7399,6 +7417,34 @@ public void testGetScopeScopeKeyObjectEntriesPage() throws Exception {
 			_siteScopedObjectEntry1.getObjectEntryId());
 	}
 
+	@Test
+	public void testGetScopeScopeKeyPageWithSiteExternalReferenceCode()
```

Comment:
Could you let me know where you got the name of this method?
The method I saw the name is `getScopeScopeKeyByExternalReferenceCode` so the name should be `testGetScopeScopeKeyByExternalReferenceCode`